scss/common/_base.scss

```diff
@@ -387,6 +386,53 @@
         opacity: .3;
     }
 
+    $loading-size: 12px !default;
```

Comment:
Yes, the $loading-size is related to the icon size. The suggested calculation sounds OK - I added it :)